AvoidAsteroids/GameScene.swift

```diff
@@ -7,39 +7,292 @@
 //
 
 import SpriteKit
+import GameplayKit
 
-class GameScene: SKScene {
+enum GameState {
+    case ShowingLogo
+    case Playing
+    case Dead
+}
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
+    var logo: SKSpriteNode!
+    var gameOver: SKSpriteNode!
+    
+    var gameState = GameState.ShowingLogo
+    
+    var player: SKSpriteNode!
+    
+    var scoreLabel: SKLabelNode!
+    
+    var score = 0 {
+        didSet {
+            scoreLabel.text = "SCORE: \(score)"
+        }
+    }
+    
+    //var backgroundMusic: SKAudioNode!
```

Comment:
Either you delete it permanently or you put it back to live. Do not comment out code